mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/MessageMapperTest.java

```diff
@@ -1217,7 +1217,7 @@ void getUidsShouldNotReturnUidsOfDeletedMessages() throws Exception {
 
         messageMapper.updateFlags(benwaInboxMailbox,
             new FlagsUpdateCalculator(new Flags(Flag.DELETED), FlagsUpdateMode.ADD),
-            MessageRange.range(message2.getUid(), message4.getUid()));
+            MessageRange.range(message2.getUid(), message4.getUid())).forEachRemaining(any -> {});
```

Comment:
Because
`messageMapper.updateFlags` return `iterator`
Add `.forEachRemaining(any -> {})` for ensuring the updateFlags is all  completed